Gemfile.lock

```diff
@@ -493,7 +493,7 @@ GEM
     responders (3.1.1)
       actionpack (>= 5.2)
       railties (>= 5.2)
-    rexml (3.2.6)
+    rexml (3.3.7)
```

Comment:
Could this gem change be causing xml_source_processing errors?

Reply:
I have encountered this before, it seems the change in rexml gem removed the extra '\n' here. 

I'm not entirely sure if this is indeed due to some gem updates they are working on at their end, but if we check most of the difference are extra whitespaces and '\n', I don't think this is fatal for our codebase. @saracarl